ansible/roles/xml-app-config/files/deployment-scripts/backend_deployment.yml

```diff
@@ -18,7 +18,6 @@
         - "BEP"
         - "EWF"
       supportscripts_file: "supportscripts"
```

Comment:
You can move `supportscripts` to the `directories` list now as it will be moved to the same final location.